src/core/Tile/Tile.tsx

```diff
@@ -147,15 +147,15 @@ export const Tile = (props: TileProps) => {
       {...rest}
     >
       {thumbnail && (
-        <div className='iui-thumbnail'>
+        <div className='iui-tile-thumbnail'>
           {typeof thumbnail === 'string' ? (
             <div
-              className='iui-picture'
+              className='iui-tile-thumbnail-picture'
               style={{ backgroundImage: `url(${thumbnail})` }}
             />
           ) : thumbnail && (thumbnail as JSX.Element).type === 'img' ? (
             React.cloneElement(thumbnail as JSX.Element, {
-              className: 'iui-picture',
+              className: 'iui-tile-thumbnail-picture',
```

Comment:
This part is also interesting.. we have example with ifram as picture and there we need to pass class `iui-tile-thumbnail-picture`.. in react it will get `iui-thumbnail-icon` instead...
Rules actually for `iui-thumbnail-icon` are also interesting in css... We have its size set if its children of thumbnail and we have some more rules, if its next to `iui-tile-thumbnail-picture` <- not even possible in react right now :/

Reply:
So what do you suggest doing? iframes are kinda an edge case so the user could just add their own CSS

Reply:
I am more wondering, why that explicit img check was added...